README.md

```diff
@@ -29,7 +34,17 @@ configure(() => {
 }, module);
 ```
 
-Note that in the above example, we're assuming the convention of each module containing a function as the `default` export. We'll account for that when writing our stories in the next section.
+or if you are using Storybook v6.
```

Comment:
should we support v6 moving forward?

Reply:
v5 and v6 are supported, right now there's no breaking change or reason to not support v5.